src/pages/api/auth/[...nextauth].ts

```diff
@@ -33,6 +35,20 @@ const preview_providers = [
       };
     },
   }),
+  // Admin (committee head) Test User
+  CredentialsProvider({
+    name: 'MERIT_COMMITTEE_HEAD',
+    id: 'admin-test-provider',
+    // @ts-ignore comment
```

Comment:
remove ts-ignore, to satisfy typescript:
- make id a string ('3')
- add another field to credentialsprovider, `credentials: {}`

Reply:
Fixed, lmk if it works